tardis/plasma/properties/property_collections.py

```diff
@@ -1,5 +1,17 @@
+from tardis.plasma.properties import (BetaRadiation, LevelBoltzmannFactor,
+    Levels, Lines, SelectedAtoms, AtomicMass, LTEPartitionFunction,
+    LevelPopulationLTE, LevelNumberDensity, PhiSahaLTE, GElectron,
+    IonizationData, NumberDensity, IonNumberDensity, LinesLowerLevelIndex,
+    LinesUpperLevelIndex, TauSobolev, TRadiative, AtomicData, Abundance,
+    Density, TimeExplosion)
 
-class LTEInputs(list):
-    pass
-
-
+class LTEProperties(list):
```

Comment:
Okay the ideas for these collections is more that you add together multiple collections. So here's my suggestion of how to do this:

```python

class PlasmaPropertyCollection(list):
    pass

generic_inputs = PlasmaPropertyCollection([AtomicData, TRadiative, ...])
lte_processing = PlasmaPropertyCollection([LevelPopulationLTE, ...])


my_lte_properties = generic_inputs + lte_processing + whatever else you need
```